.github/workflows/coverage.yaml

```diff
@@ -0,0 +1,79 @@
+name: Test Coverage
+on:
+  workflow_dispatch:
+  workflow_call:
+  schedule:
+    # every day at 9am PST
+    - cron: "0 16 * * *"
+  pull_request:
+    types: [labeled, opened, synchronize, reopened]
+
+env:
+  CARGO_INCREMENTAL: "0"
+  CARGO_TERM_COLOR: always
+
+# cancel redundant builds
+concurrency:
+  # cancel redundant builds on PRs (only on PR, not on branches)
+  group: ${{ github.workflow }}-${{ (github.event_name == 'pull_request' && github.ref) || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  rust-unit:
+    if: github.event_name != 'pull_request' || contains(github.event.pull_request.labels.*.name, 'CICD:run-coverage') 
+    runs-on: high-perf-docker
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # get all the history because cargo xtest --change-since origin/main requires it.
+      - uses: ./.github/actions/rust-setup
+      - run: rustup component add llvm-tools-preview
+      - uses: taiki-e/install-action@v1.5.6
+        with:
+          tool: nextest,cargo-llvm-cov
+      - run: docker run --detach -p 5432:5432 cimg/postgres:14.2
+      - run: cargo llvm-cov nextest --workspace --lcov --output-path lcov.info --exclude smoke-test --exclude backup-cli --exclude testcases -- --profile ci
```

Comment:
TIL llvm-cov, I was using RUSTFLAGS=-Cinstrument-coverage lol

Reply:
@msmouse was there a reason why backup-cli test is excluded?

Reply:
we should re-include it, there was something broken with how it was running on circle

Reply:
> llvm-cov

I don't quite understand what you mean?
If you go to he codecov link it says the coverage is ~35%.

Reply:
can we do --no-fail-fast like before?